packages/client/lib/__tests__/IT.spec.ts

```diff
@@ -222,164 +287,36 @@ describe('OID4VCI-Client should', () => {
-            length: preAuthorizedCode.length,
+      expect(credentialOffer.original_credential_offer).toEqual({
+        credential_configuration_ids: ['OpenBadgeCredentialUrl'],
+        credential_issuer: ISSUER_URL,
+        grants: {
+          'urn:ietf:params:oauth:grant-type:pre-authorized_code': {
+            'pre-authorized_code': "oaKazRN8I0IbtZ0C7JuMn5",
+            tx_code: {
+              description: 'Please enter the serial number of your physical drivers license',
+              input_mode: 'text',
+              length: 22,
+            },
           },
-        },
+        }
       });
 
       nock(ISSUER_URL)
-        .post(/token.*/)
-        .reply(200, JSON.stringify(mockedAccessTokenResponse));
+      .post(/token.*/)
+      .reply(200, JSON.stringify(mockedAccessTokenResponse));
 
       /* The actual access token calls */
       const accessTokenClient: AccessTokenClient = new AccessTokenClient();
       const accessTokenResponse = await accessTokenClient.acquireAccessToken({ credentialOffer: credentialOffer, pin: '1234' });
       expect(accessTokenResponse.successBody).toEqual(mockedAccessTokenResponse);
       // Get the credential
-      nock(ISSUER_URL)
-        .post(/credential/)
-        .reply(200, {
-          format: 'jwt-vc',
-          credential: mockedVC,
-        });
-      const credReqClient = CredentialRequestClientBuilder.fromCredentialOffer({ credentialOffer: credentialOffer })
-        .withFormat('jwt_vc')
-
-        .withTokenFromResponse(accessTokenResponse.successBody!)
-        .build();
-
-      //TS2322: Type '(args: ProofOfPossessionCallbackArgs) => Promise<string>'
-      // is not assignable to type 'ProofOfPossessionCallback'.
-      // Types of parameters 'args' and 'args' are incompatible.
-      // Property 'kid' is missing in type '{ header: unknown; payload: unknown; }' but required in type 'ProofOfPossessionCallbackArgs'.
-      const proof: ProofOfPossession = await ProofOfPossessionBuilder.fromJwt({
-        jwt,
-        callbacks: {
-          signCallback: proofOfPossessionCallbackFunction,
-        },
-        version: OpenId4VCIVersion.VER_1_0_11,
-      })
-        .withEndpointMetadata({
-          issuer: 'https://issuer.research.identiproof.io',
-          credential_endpoint: 'https://issuer.research.identiproof.io/credential',
-          token_endpoint: 'https://issuer.research.identiproof.io/token',
-        })
-        .withKid('did:example:ebfeb1f712ebc6f1c276e12ec21/keys/1')
-        .build();
-      const credResponse = await credReqClient.acquireCredentialsUsingProof({
-        proofInput: proof,
-        credentialTypes: credentialOffer.original_credential_offer.credential_configuration_ids[0],
-      });
-      expect(credResponse.successBody?.credential).toEqual(mockedVC);
-    },
-    UNIT_TEST_TIMEOUT,
-  );
-});
-
-describe('OIDVCI-Client for v1_0_13 should', () => {
```

Comment:
Why is this removed?

Reply:
added back. not sure why it's not showing it here